### PR TITLE
add build arg

### PR DESCRIPTION
**Pull request by ccl0326**
Add build arg in build stage, we can use docker arg syntax into lain cli to pass gitlab token and so on.

### CODE REVIEW

**Comment by ericpai** [comment]
This will be merged at v2.3.0. Thanks for your contribution.

File: lain_sdk/yaml/conf.py [hunk 1]
```diff
@@ -12,4 +12,4 @@
 
 PRIVATE_REGISTRY = None if etc is None else etc.get('private_docker_registry', None)
 
-DOMAIN = None if etc is None else etc.get('DOMAIN', 'lain.local')
+DOMAIN = None if etc is None else etc.get('domain', 'lain.local')
```
Comment (wchaoyi) [inline lain_sdk/yaml/conf.py]: I think there is no need to change the name, at least ensure that the old usage is compatible。

Comment (ccl0326) [inline lain_sdk/yaml/conf.py]: fixed and rebase.

Comment (wchaoyi) [inline lain_sdk/yaml/conf.py]: still have conflicts and could you rebase on the master?

Comment (ccl0326) [inline lain_sdk/yaml/conf.py]: rebased on master already.

**Pull request merged**